internal/driver/driver.go

```diff
@@ -878,6 +942,16 @@ func (d *Driver) newDevice(name string, protocols map[string]models.ProtocolProp
 	}, nil
 }
 
+func (d *Driver) updateTranscoder(device *Device, fdPath string) error {
```

Comment:
maybe rename to `updateTranscoderInputPath`. Also, would this make more sense as something inside of the `*Device` struct itself?

Reply:
Do you mean add the path as a entry in the struct or to make this a device method?